prompts/deepset-conversational-agent-without-tools.yaml

```diff
@@ -0,0 +1,11 @@
+name: deepset/conversational-agent-without-tools
+text: |
+  The following is a conversation between a human and an AI.\n{memory}\nHuman: {query}\nAI:
```

Comment:
I think memory is history, no? Either way, it should go at the end of the prompt IIRC? Double check please

Reply:
I updated the variable name into `memory` with this [PR](https://github.com/deepset-ai/haystack/pull/5203). 
I didn't understand how {memory} should go at the end of the prompt 🤔